### PR TITLE
Add blog post announcing Trino Summit lineup

### PR DESCRIPTION
**Pull request by colebow**

### CODE REVIEW

File: _posts/2023-11-21-trino-summit-2023-nears-lineup.md [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+author: "Cole Bowden"
+excerpt_separator: <!--more-->
+image: /assets/blog/trino-summit-2023/lineup-blog-banner.png
+---
+
+As winter nears, the days may be getting shorter, but so is the wait until
+Trino Summit 2023! It'll be here before you know it on December 13th and 14th,
+and we've got a packed speaker lineup full of exciting talks that we're ready to
+share with the Trino community today. Read on for a preview of all the talks
+we're looking forward to, and if you're interested in attending, make sure to...
```
Comment (mosabua) [inline _posts/2023-11-21-trino-summit-2023-nears-lineup.md:13]:
```suggestion
, and if you're interested in attending, make sure to ...
```

File: _posts/2023-11-21-trino-summit-2023-nears-lineup.md [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+    </a>
+</div>
+<div class="spacer-30"></div>
+
+<!--more-->
+
+So, who's going to be talking at Trino Summit? Here's a quick rundown of the
+talks coming in from various companies.
+
+## Starburst
```
Comment (mosabua) [inline _posts/2023-11-21-trino-summit-2023-nears-lineup.md:27]: Lets break this up along the topic and company .. not just the company .. and leave anything that is not confirmed out

File: _posts/2023-11-21-trino-summit-2023-nears-lineup.md [hunk 3]
```diff
@@ -0,0 +1,189 @@
...
+
+As always, our keynote will come from Martin Traverso, Trino co-founder and
+co-CTO at Starburst. He'll be giving a project update on everything exciting
+that's happened in Trino since Trino Fest, as well as a sneak peek at the
+roadmap for features coming to Trino in 2024. It's one of the best ways to keep
+up with the ongoing developments in the Trino community, and you won't want to
+miss it.
+
+A second talk from Starburst will be exploring the new Trino Gateway, a proxy
```
Comment (mosabua) [inline _posts/2023-11-21-trino-summit-2023-nears-lineup.md:36]: This talk should be mentioning Bloomberg, Starburst and Naver at least .. maybe also Lyft

File: _posts/2023-11-21-trino-summit-2023-nears-lineup.md [hunk 4]
```diff
@@ -0,0 +1,189 @@
...
+
+Bazaar's talk will discuss how they leverage Trino's capabilities to optimize
+data analysis and support data-driven decision-making. The talk specifically
+explores including real-time data querying across multiple sources and
+performance optimization, illustrating Trino's role in Bazaar's data-centric
+strategies. This presentation provides in-depth insights for individuals
+well-versed in Trino, shedding light on the platform's transformative impact on
+enhancing e-commerce operations.
+
+## Superset
```
Comment (mosabua) [inline _posts/2023-11-21-trino-summit-2023-nears-lineup.md:174]: Company is Preset

**Review by mosabua — CHANGES_REQUESTED** [review]
I will review more once it is reduced to the ones we are actually adding and the section titles are updated

File: _posts/2023-11-21-trino-summit-2023-nears-lineup.md (hunk 1 above)
Comment (colebow) [reply _posts/2023-11-21-trino-summit-2023-nears-lineup.md:13]: The space before the ellipsis isn't necessarily correct. Every style guide differs on its recommendations, but I've found the lack of space before trailing off to be the most commonly-accepted approach in informal writing. See, for example, the [Buzzfeed Style Guide](https://www.buzzfeed.com/buzzfeednews/buzzfeed-style-guide).

File: _posts/2023-11-21-trino-summit-2023-nears-lineup.md [hunk 5]
```diff
@@ -0,0 +1,174 @@
...
+for ordinary users and analysts to distribute large amounts of unstructured
+data. With the power of a custom-built polymorphic table function,
+they were able to invoke Python code within Trino to help structure that data
+for analysis, solving the problem in a powerful and fascinating way. We'll get
+to hear about polymorphic table functions, how they work in Trino, and how
+anyone else may be able to leverage them to solve problems.
+
+## Raft: Avoiding pitfalls with query federation in data lakehouses
+
+Raft has partnered with the DoD to build a Data Fabric that is built on top of
```
Comment (mosabua) [inline _posts/2023-11-21-trino-summit-2023-nears-lineup.md:122]: Probably need to say "US Department of Defense"

Comment (mosabua) [reply _posts/2023-11-21-trino-summit-2023-nears-lineup.md:122]: also "data fabric"

File: _posts/2023-11-21-trino-summit-2023-nears-lineup.md [hunk 6]
```diff
@@ -0,0 +1,174 @@
...
+Delta Lake, Trino, Apache Kafka, and Open Policy Agent (OPA). This talk will
+discuss the challenges involved, provide solutions and considerations for each,
+and end with a demo of Raft’s Data Fabric. The talk will focus on
+a plugin for Trino, developed by Raft, that uses OPA as a policy engine to
+provide fine-grained access control at query time based on a user’s JWT passed
+along with the query.
+
+## Treasure Data: Secure exchange SQL
+
+Secure Exchange SQL is a production Data Clean Room service deployed at Treasure
```
Comment (mosabua) [inline _posts/2023-11-21-trino-summit-2023-nears-lineup.md:132]:
```suggestion
Secure Exchange SQL is a production data clean room service deployed at Treasure
```

File: _posts/2023-11-21-trino-summit-2023-nears-lineup.md [hunk 7]
```diff
@@ -0,0 +1,174 @@
...
+Data, which leverages Trino and differential privacy technology to enable
+cross-company data analysis while mitigating the risk of privacy breaches.
+In their session, they'll introduce the concept of differential privacy and
+discuss the privacy protection methods that need to be implemented during SQL
+processing. To minimize changes to Trino's codebase, they employed approaches of
+SQL rewriting and validation at the logical plan level. They'll explain these
+methods and provide some practical use cases of their Data Clean Room.
```
Comment (mosabua) [inline _posts/2023-11-21-trino-summit-2023-nears-lineup.md:139]:
```suggestion
methods and provide some practical use cases of their data clean room.
```

File: _posts/2023-11-21-trino-summit-2023-nears-lineup.md [hunk 8]
```diff
@@ -0,0 +1,174 @@
...
+performance optimization, illustrating Trino's role in Bazaar's data-centric
+strategies. This presentation provides in-depth insights for individuals
+well-versed in Trino, shedding light on the platform's transformative impact on
+enhancing e-commerce operations.
+
+## Preset: Visualizing Trino with Superset
+
+Preset will be diving into the "last mile" of the modern data stack and
+show you how to query and visualize data pulled from Trino with Apache Superset
+and/or Preset. Specifically, they'll discuss into things like Trino's federated
```
Comment (mosabua) [inline _posts/2023-11-21-trino-summit-2023-nears-lineup.md:163]:
```suggestion
and/or Preset. Specifically, they'll discuss things like Trino's federated
```

File: _posts/2023-11-21-trino-summit-2023-nears-lineup.md [hunk 9]
```diff
@@ -0,0 +1,174 @@
...
+query support (a common wish for Superset users) and how Superset can support
+near-real-time analytics for Trino users. They'll also give a demo of connecting
+to Trino, building SQL Queries, designing charts and dashboards, and other ways
```
Comment (mosabua) [inline _posts/2023-11-21-trino-summit-2023-nears-lineup.md:166]:
```suggestion
to Trino, building SQL queries, designing charts and dashboards, and other ways
```

File: _posts/2023-11-21-trino-summit-2023-nears-lineup.md [hunk 10]
```diff
@@ -0,0 +1,174 @@
...
+to gain insight and stay on top of your data.
+
+## VAST: The VAST database catalog
+
+The VAST Database connector for Trino was open-sourced this year! They'll be
+discussing the architecture of VAST and the connector, the purpose and major use
+cases for it, and demonstrate the workflows surrounding the VAST Database in the
+Trino ecosystem.
```
Comment (mosabua) [inline _posts/2023-11-21-trino-summit-2023-nears-lineup.md:174]: Blog post needs a conclusion ... something like .. more sessions will be announced and there are already great so dont forget to register and see you there.

**Review by mosabua — CHANGES_REQUESTED** [review]

File: _posts/2023-11-21-trino-summit-2023-nears-lineup.md [hunk 11]
```diff
@@ -0,0 +1,174 @@
...
+LinkedIn has been keeping up with Trino releases as an impressive rate, but
+getting to that point has required a lot of time, effort, and work on
+streamlining the update process. They'll be discussing the challenges of
+breaking changes, applying internal patches, and ensuring that there are no
+meaningful performance regressions. They've automated much of this, including
+implementing a post-commit integration test suite that ensures nothing has
+broken, and creating an automated test framework that can validate the
+performance of each new Trino release before it deploys to users.
+
+## EA: Migrating 120 million HMS metadata records to AWS Glue without customer impact
```
[the remark this answers is not in the document]
Comment (colebow) [reply _posts/2023-11-21-trino-summit-2023-nears-lineup.md:89]: I think they're more recognizable as EA than as Electronic Arts.

File: _posts/2023-11-21-trino-summit-2023-nears-lineup.md (hunk 1 above)
Comment (mosabua) [reply _posts/2023-11-21-trino-summit-2023-nears-lineup.md]: no worries

File: _posts/2023-11-21-trino-summit-2023-nears-lineup.md (hunk 11 above)
[the remark this answers is not in the document]
Comment (mosabua) [reply _posts/2023-11-21-trino-summit-2023-nears-lineup.md]: ok ..

**Review by mosabua — APPROVED** [review]
Looks great now. Lets ship and announce!

**Pull request merged**